api/models/graphite.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/grafana/metrictank/idx"
 	pickle "github.com/kisielk/og-rek"
 	opentracing "github.com/opentracing/opentracing-go"
-	"gopkg.in/macaron.v1"
+	macaron "gopkg.in/macaron.v1"
```

Comment:
vscode in the default settings uses `goreturns` to format the code on saving. that's documented here: https://github.com/Microsoft/vscode-go/wiki/On-Save-features#format-on-save
if you want I can disable that, or switch it to `gofmt` which doesn't add the import name explicitly even if the package name differs from the import path.

Reply:
i don't think goreturns is to blame here, something else is going on
```
go get -u github.com/sqs/goreturns && goreturns -w api/routes.go && grep macaron.v1 api/routes.go

	"gopkg.in/macaron.v1"
```

Reply:
that's strange, because i've tested that too before I wrote the last comment:

```
mst@mst-nb1:~/documents/code/go/src/github.com/grafana/metrictank$ cat api/models/graphite.go | grep macaron.v1
        "gopkg.in/macaron.v1"
mst@mst-nb1:~/documents/code/go/src/github.com/grafana/metrictank$ gofmt api/models/graphite.go | grep macaron.v1
        "gopkg.in/macaron.v1"
mst@mst-nb1:~/documents/code/go/src/github.com/grafana/metrictank$ goreturns api/models/graphite.go | grep macaron.v1
        macaron "gopkg.in/macaron.v1"
```

Reply:
after doing `go get -u github.com/sqs/goreturns` it doesn't do that anymore. i guess something must have changed in the newer version. anyway, i'll undo those changes that the old version made